### PR TITLE
Required/tasks 2

### PR DESCRIPTION
**Pull request by eduardourso**
#### TL;DR

Calculator task 2 of the `Required Tasks` in the Programming Project 1.
#### Problem

Calculator didn't have the floating number functionality.
#### Solution
- Updated protocol as demonstrated in lecture 3 to return tuple type.
- Implemented function that allows user to add floating numbers to the operations
#### Test(s) added

NO.
#### Paired with

just me

### CODE REVIEW

File: Calculator.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1,1055 +1,580 @@
...
+		6BCFAF341CCECC7A000A654C /* Build configuration list for PBXNativeTarget "CalculatorTests" */ = {
+			isa = XCConfigurationList;
+			buildConfigurations = (
+				6BCFAF351CCECC7A000A654C /* Debug */,
+				6BCFAF361CCECC7A000A654C /* Release */,
+			);
+			defaultConfigurationIsVisible = 0;
+			defaultConfigurationName = Release;
+		};
+/* End XCConfigurationList section */
```
Comment (amlcurran) [inline Calculator.xcodeproj/project.pbxproj:577]: man, your Xcode project file likes changing 😆

Comment (eduardourso) [inline Calculator.xcodeproj/project.pbxproj:577]: Unexpected change 😲

File: Calculator/ViewModel/CalculatorViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,95 @@
+import Foundation
+
+struct CalculatorViewModel : CalculatorOperationsProtocol {
```
Comment (amlcurran) [inline Calculator/ViewModel/CalculatorViewModel.swift]: given all the functions here are mutating, perhaps a class could be better than a struct?

Comment (eduardourso) [inline Calculator/ViewModel/CalculatorViewModel.swift]: Makes total sense. Will refactor it.

**Comment by amlcurran** [comment]
👍 lets merge this and see what we can refactor later!

**Pull request merged**